src/pint/models/stand_alone_psr_binaries/ELL1_model.py

```diff
@@ -46,10 +46,10 @@ def ttasc(self):
         """
         ttasc = t - TASC
         """
+        t = self.t
         if not hasattr(self.t, "unit") or self.t.unit == None:
             t = self.t * u.day
-        t = self.t
-        ttasc = (t - self.TASC).to("second")
+        ttasc = (t - self.TASC.value * u.day).to("second")
```

Comment:
Why this change? Is it to ensure you get a Quantity with units of days, rather than a `Time()`?

Reply:
Yeah, the problem wasn't with .tt0, it was with the fact that self.t is always a numpy array, and I don't think we want that to be an array of Times (since this is a standalone module).  If self.t is not made of Times, then we can't subtract a Time from them.